setup.py

```diff
@@ -16,7 +16,7 @@
 data_files_path = tools_path.relative_to(base_prefix).parents[1]
 
 setuptools.setup(
-    name="ts_hexapod",
+    name="ts_mthexapod",
     description="CSC for main telescope hexapods (camera and secondary mirror)",
     use_scm_version={
         "write_to": "python/lsst/ts/hexapod/version.py",
```

Comment:
Shouldn't that be python/lsst/ts/mthexapod/version.py ?

Reply:
Good catch. And I may have missed the change to the bin scripts a few lines later. In any case it's all fixed now.